dm/ui/src/models/task.ts

```diff
@@ -166,8 +172,12 @@ export type DmapiStartTaskApiArg = {
   taskName: string
   startTaskRequest?: {
     remove_meta?: boolean
-    safe_mode_time_duration?: any
+    safe_mode_time_duration?: string
     source_name_list?: string[]
+
+    /**
+     * RFC3339 format date string
```

Comment:
Maybe we need to record format is like `2006-01-02T15:04:05` or `2006-01-02T15:04:05` and will give UTC time value to backend.

Reply:
sure